queue/queue.py

```diff
@@ -8,10 +8,19 @@ def __init__(self):
     self.storage = LinkedList()
 
   def enqueue(self, item):
-    pass
-  
+    if item == None:
+      return 0
+    elif self.storage.get_length == 0:
+      return 0
+    self.storage.add_to_tail(item)
+    
   def dequeue(self):
-    pass
+    return self.storage.remove_head()
 
   def len(self):
-    pass
+    count = 0
+    counter = self.storage.head
+    while counter:
```

Comment:
you could add and remove from `self.size` every time you enqueue or dequeue.  You wouldn't need this loop then.